src/util/editorCommentUtils.js

```diff
@@ -0,0 +1,42 @@
+import { v4 as uuid } from "uuid";
+import { Editor } from 'slate'
+
+const COMMENT_THREAD_PREFIX = "commentThread_";
+
+export function getMarkForCommentThreadID(threadID) {
```

Comment:
Can you add some comments on how to use these apis? (For ex: what is "mark" for a comment)

Reply:
I grabbed this from Emily's code so I don't?? actually know?? but I will attempt to document it further yes. Most of this stuff is explained by the slate documentation, I'll link the relevant parts in the comments